### DISCUSSION TITLE
how to run the epg-grabber command automatically

### DISCUSSION THREAD

**Discussion by kareemelmasry** [iptv-org/community#473] [Q&A]
### Description

Hiii.

how to run the npx epg-grabber --config=sites/bein.com/bein.com.config.js --channels=sites/bein.com/bein.com.channels.xml --output=BeinSports.xml command automatically every 4 hours ?
by windows task schedule or any other way please.

your help is highly appreciated

**Comment by PopeyeTheSai10r** [comment]
You can try [Node Schedule](https://www.npmjs.com/package/node-schedule) since you are already using node to run the epg-grabber.
Or take a look at [Run a Node script with Windows task scheduler](https://www.dev-tips-and-tricks.com/run-a-node-script-with-windows-task-scheduler) and [Using Windows Task Scheduler to Automate NodeJS Scripts](https://joshuatz.com/posts/2020/using-windows-task-scheduler-to-automate-nodejs-scripts/)

**Reply by Mauricio300808** [reply]
Has anyone been able to make this work? to run the grab using either node schedule or toad schedule or even the task scheduler... im kind of new to node and npm so any help or suggestions or examples on how to write the script are appreciated!

**Comment by freearhey** [comment]
https://github.com/iptv-org/epg?tab=readme-ov-file#run-on-schedule
